app/javascript/packs/components/RangeOption.jsx

```diff
@@ -17,8 +17,9 @@ class RangeOption extends React.Component {
       this.props.onSave(id, num);
     }, PAUSE_INTERVAL_BEFORE_SAVE);
 
+    // Check if dependent questions should be triggered initially
     if (this.props.answer) {
-      this.onChange(this.props.answer.text);
+      this.props.onChange(this.props.answer.text);
```

Comment:
Should this include `this.props.answer.selected_option_id` as the first parameter? I think otherwise, we may not be setting `text` within store on load.